app/controllers/api/v1/apps/credentials_controller.rb

```diff
@@ -4,6 +4,6 @@ class Api::V1::Apps::CredentialsController < Api::BaseController
   def show
     return doorkeeper_render_error unless valid_doorkeeper_token?
 
-    render json: doorkeeper_token.application, serializer: REST::ApplicationSerializer, fields: %i(name website vapid_key client_id scopes)
+    render json: doorkeeper_token.application, serializer: REST::ApplicationSerializer, fields: %i(name website vapid_key client_id scopes redirect_uris)
```

Comment:
@ClearlyClaire is there a way to write this to "exclude" certain fields? Because that's what we really want here, instead of "include" certain fields. (we're excluding `client_secret`)

Reply:
I don't think so. At least I haven't found one.

Reply:
I think the current version is good enough, honestly. Though that suggestion isn't too bad either.

Reply:
I've done that split now since it was easy to do, we now have `Application` for verify and `CredentialApplication` for registration.